tests/tools/TestApp/rx_app.c

```diff
@@ -2,25 +2,55 @@
 #include "Inc/input.h"
 #include "Inc/mcm.h"
 #include <unistd.h>
+#include <stdlib.h>
 
 const char* client_cfg;
 const char* conn_cfg;
 
 int main(int argc, char* argv[]){
+    if(!is_root()){
```

Comment:
```suggestion
    if (!is_root()) {
```
Don't miss the spaces.

Reply:
run clan-formater on this code